logic/scorer/scorer.go

```diff
@@ -87,29 +103,127 @@ func scoreFifteens(ptVals []int) (scoreType, int) {
 	return st, int(numFifteens * 2)
 }
 
-func howManyAddUpTo(goal int, ptVals []int) uint {
-	if len(ptVals) == 0 {
+func howManyAddUpTo(goal int, ptVals values, start int) uint {
+	if start == cardsPerHand {
 		return 0
 	}
 
 	var many uint
-	for i, o := range ptVals {
+	for i := start; i < cardsPerHand; i++ {
```

Comment:
:cactus: `for i := start; i < len(ptVals); i++ {`

Reply:
is there benefit to using `len` when we _know_ it will evaluate to `cardsPerHand` (because the type is defined as such)?

Reply:
the perceived benefit is that the intent of this for loop is to go through `ptVals`. If I see that the iteration stops at `len(ptVals)`, then on my first pass reading, I immediately understand what is happening